Gemfile

```diff
@@ -42,6 +42,10 @@ group :bench do
 end
 
 group :test do
+  # allows for rspec-like invocation of single tests
+  # e.g.: $ m test/path.rb:lineNum
+  gem 'm', '~> 1.4.2'
```

Comment:
well, if we add it, we want to exclude it on CI.. one less thing to build.  and I haven't been using it lately. .ruby -Ilib:test path/to/file_test.rb